### PR TITLE
Ci check translations updates

### PR DESCRIPTION
**Pull request by JackHowa**
- All of the current lokalise changes

### CODE REVIEW

File: blocks/full-author-bio-block/intl.json [hunk 1]
```diff
@@ -11,157 +11,157 @@
 		"sv": "Följ"
 	},
 	"full-author-bio-block.social-email": {
-		"de": "E-Mail",
+		"de": "Email",
```
Comment (JackHowa) [inline blocks/full-author-bio-block/intl.json:14]: @nschubach 's pr should fix this full author bio block being removed

File: locale/de.json [hunk 2]
```diff
@@ -22,7 +20,24 @@
 		"global.social-pinterest": "Pinterest",
 		"global.social-email": "E-Mail",
 		"global.opens-in-new-window": "(Wird in einem neuen Fenster geöffnet)",
-		"global.sponsored-content": "Werblicher Inhalt"
+		"global.sponsored-content": "Werblicher Inhalt",
+		"global.gallery-text": "Gallery",
+		"global.video-text": "Video",
+		"global.social-email-content": "Send an email to %{authorName}",
+		"global.social-rss": "Subscribe to %{authorName} RSS feed",
+		"global.social-soundcloud-content": "Listen to %{authorName} on SoundCloud",
```
Comment (matthewroach) [inline locale/de.json:28]: We've now lost our translations?

Comment (JackHowa) [reply locale/de.json:28]: it looks like it? I'm not sure if these have been migrated in @malavikakoppula 's pr

Comment (JackHowa) [reply locale/de.json:28]: maybe the keys there point to the correct ones

Comment (JackHowa) [reply locale/de.json:28]: fixed in lokalise https://github.com/WPMedia/arc-themes-blocks/pull/1499

**Comment by JackHowa** [comment]
fixed now in #1499

**Pull request closed without merging**